pkg/features/kube_features.go

```diff
@@ -1066,6 +1073,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	LegacyServiceAccountTokenCleanUp: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // GA in 1.30; remove in 1.32
 
+	LegacyNodeRunOnceMode: {Default: true, PreRelease: featuregate.Alpha},
```

Comment:
See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/feature-gates.md#deprecation

It describes how to deprecate with gates (though in this case you want to start with it enabled, to satisfy deprecation policy)

Reply:
I would leave a TODO here to change default to false in or after 1.34

Reply:
> See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/feature-gates.md#deprecation
> 
> It describes how to deprecate with gates (though in this case you want to start with it enabled, to satisfy deprecation policy)

Thanks for the reminder. 

> I would leave a TODO here to change default to false in or after 1.34

Since the KEP describes that it will be set to false by default in v1.32, I have added comment according to the KEP.

Reply:
The deprecation policy is a little vague about this.  It says "Deprecated behaviors must function for no less than 1 year after their announced deprecation."  Do we think that "function" means "by default" or "with effort" ?

My read of it was "by default"

Reply:
All right.